includes/wikia/models/PromoImage.class.php

```diff
@@ -76,7 +50,7 @@ public function setCityId($cityId) {
 		return $this;
 	}
 
-	public function getCityId(){
+	private function getCityId(){
 		if (empty($this->cityId) and !empty($this->dbName)){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace "and" with "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2010)